lib/Sema/ConstraintGraph.cpp

```diff
@@ -680,8 +680,34 @@ bool ConstraintGraph::contractEdges() {
         // it's use in the body. If parameter declaration is bound to
         // `inout` it's use has to be bound to `l-value`, which can't
         // happen once equivalence classes of parameter and argument are merged.
```

Comment:
Can you update the comment above, perhaps just using the comment below?

Reply:
Sure, I can remove that, although this comment is still valid for the general case.